fusor-ember-cli/app/routes/subscriptions/select-subscriptions.js

```diff
@@ -62,14 +63,10 @@ export default Ember.Route.extend({
           controller.set('subscriptionEntitlements', Ember.A(results[0]));
           controller.set('subscriptionPools', Ember.A(results[1]));
           return controller.set('isLoading', false);
-        }, function() {
-             self.modelFor('subscriptions').set('isAuthenticated');
+        }, function(error) {
              self.modelFor('subscriptions').save().then(function() {
-               self.controllerFor('subscriptions.credentials').setProperties({
-                                                                   'showErrorMessage': true,
-                                                                   'errorMsg': 'You are not currently logged in. Please log in below.'
-                                                                 });
-               return self.transitionTo('subscriptions.credentials');
```

Comment:
I previously logged out the user and auto-redirected to the credentials form if there was an error.